packages/components/src/circular-option-picker/style.scss

```diff
@@ -69,7 +69,7 @@ $color-palette-circle-spacing: 12px;
 
 	&:hover {
 		// Override default button hover style.
-		box-shadow: inset 0 0 0 ($color-palette-circle-size * 0.5) !important;
```

Comment:
This could be a breaking change for the component. It's annoying it's there in the first place, but it'd be nice to find an alternative to updating this one.